test/Microsoft.Azure.Devices.Edge.Test.TempSensor/Test.cs

```diff
@@ -11,6 +12,10 @@ namespace Microsoft.Azure.Devices.Edge.Test.TempSensor
 
     public class Test
     {
+        IEdgeDaemon CreateEdgeDaemon(string installerPath) => RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
```

Comment:
nit: extract this as a helper method in order to use in other tests?

Reply:
What other tests? :)  My next PR introduces a second test, so that where I'm doing this kind of work.